frontend/src/components/TemplateEditor/controls/__snapshots__/ControlPanelTextInput.test.js.snap

```diff
@@ -54,6 +54,7 @@ exports[`ControlPanelTextInput component renders as expected 1`] = `
             data-ouia-safe="true"
             data-testid="text-controlId"
             id="controlId"
+            placeholder="Enter creation.app.name"
```

Comment:
Same comment here as for ControlPanelSingleSelect.

Reply:
Not an issue - translation is done at a higher level and does not take effect when you instantiate controls directly.